starters/cms/package.json

```diff
@@ -1,34 +1,39 @@
 {
     "name": "cms-starter",
-    "private": true,
     "version": "0.0.0",
+    "private": true,
     "type": "module",
     "scripts": {
-        "dev": "vite",
         "build": "vite build",
-        "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+        "check": "npm run typecheck && npm run lint",
+        "dev": "vite",
+        "format": "prettier . --write",
+        "lint": "eslint . --max-warnings 0",
+        "pack": "framer-plugin-tools pack",
         "preview": "vite preview",
-        "pack": "npx framer-plugin-tools@latest pack"
+        "typecheck": "tsc"
     },
     "dependencies": {
-        "framer-plugin": "^3.0.0-beta.0",
-        "react": "^18",
-        "react-dom": "^18",
-        "vite-plugin-mkcert": "^1"
+        "framer-plugin": "^3.0.0-beta.1",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1"
     },
     "devDependencies": {
-        "@eslint/js": "^9",
-        "@types/react": "^18",
-        "@types/react-dom": "^18",
-        "@vitejs/plugin-react": "^4.3.1",
-        "@vitejs/plugin-react-swc": "^3",
-        "eslint": "^9.9.0",
-        "eslint-plugin-react-hooks": "^5.1.0",
-        "eslint-plugin-react-refresh": "^0.4.9",
-        "globals": "^15.9.0",
-        "typescript": "^5.3",
-        "typescript-eslint": "^8.0.1",
-        "vite": "^6",
-        "vite-plugin-framer": "^1"
+        "@eslint/js": "^9.21.0",
+        "@types/react": "^18.3.18",
+        "@types/react-dom": "^18.3.5",
+        "@vitejs/plugin-react": "^4.3.4",
+        "@vitejs/plugin-react-swc": "^3.8.0",
+        "eslint": "^9.21.0",
+        "eslint-plugin-react-hooks": "^5.2.0",
+        "eslint-plugin-react-refresh": "^0.4.19",
+        "framer-plugin-tools": "^1.0.0",
+        "globals": "^16.0.0",
+        "prettier": "^3.5.2",
+        "typescript": "^5.7.3",
+        "typescript-eslint": "^8.25.0",
+        "vite": "^6.2.0",
+        "vite-plugin-framer": "^1.0.6",
+        "vite-plugin-mkcert": "^1.17.7"
```

Comment:
single numbers!! look at nextjs for example

Reply:
Are you talking about this? 😏 https://github.com/vercel/next.js/commit/4ec443931aaf53bd831fa57ac5733b97d7dac550 So, they do that for templates, which don't even have their own `package.json`, but I think our starters are more like their examples, which **do** use specific versions, as they're full-fledged projects:

https://github.com/vercel/next.js/blob/canary/examples/cms-contentful/package.json

My rationale is mentioned in the commit message:

> Use full versions for dependencies as those are the minimum versions that the starter is actually confirmed to work with.

Reply:
To expand a little, imagine, we base this starter on `framer-plugin@3.1.0`, which has a `framer.doFoo()` method, added in `3.1.0`. But then somebody uses this starter in their Framer plugin monorepo with other plugins which are on `framer-plugin@3.0.0`. If the version in this `package.json` is `^3`, their package managed will just keep `framer-plugin` on `3.0.0` and the starter will break, since there's no `framer.doFoo()`. If the version in `package.json` is `^3.1.0`, the package manager will ensure that at least the starter gets `framer-plugin@3.1.0`, if not the whole repository (depends on the manager and its settings).

Reply:
Ok good points! Lets go for it as you have it then